StripePaymentSheet/StripePaymentSheet/Resources/Localizations/en.lproj/Localizable.strings

```diff
@@ -1,3 +1,6 @@
+/* Content for alert popup prompting to confirm removing a saved card. {card brand} •••{last 4} e.g. 'Visa •••3155' */
+"%1$@ •••%2$@" = "%1$@ •••%2$@";
```

Comment:
Can we double check w/ @jarvis-stripe that this should be 3 dots and not 4?  I think it's usually 4.

Reply:
Yeah, I thought that was intentional, but now second guessing. I pinged @jarvis-stripe but for now changed to 4 bullet points to match the rest of our UI.

Reply:
Confirmed with @jarvis-stripe. It should be 4 bullets, space, then last 4.